sdk/cosmos/azure-cosmos-spark-account-data-resolver-sample/sample/AccountTokenResolverSample.ipynb

```diff
@@ -5,43 +5,48 @@
    "execution_count": 0,
    "metadata": {
     "application/vnd.databricks.v1+cell": {
-     "cellmetadata": {
-      "bytelimit": 2048000,
-      "rowlimit": 10000
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
      },
-     "inputwidgets": {},
+     "inputWidgets": {},
      "nuid": "edf2bb67-c3fa-459c-bb4b-83dec2075401",
-     "showtitle": false,
+     "showTitle": false,
      "title": ""
     }
    },
    "outputs": [],
    "source": [
-    "cosmosendpoint = \"https://fabianm-oltp-spark-workshop-cdb.documents.azure.com:443/\"\n",
-    "cosmosmasterkey = \"\"\n",
-    "cosmosserviceprincipalpassword=\"\"\n"
+    "cosmosendpoint = \"<YourEndpoint>\"\n",
+    "cosmosmasterkey = \"<YourKey>\"\n",
+    "cosmosserviceprincipalpassword=\"\"\n",
+    "accountDataResolverName = \"com.azure.cosmos.spark.samples.MasterKeyAccountDataResolver\"\n",
+    "#accountDataResolverName = \"com.azure.cosmos.spark.samples.ServicePrincipalAccountDataResolver\"\n",
+    "#accountDataResolverName = \"com.azure.cosmos.spark.samples.ManagedIdentityAccountDataResolver\""
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 0,
    "metadata": {
     "application/vnd.databricks.v1+cell": {
-     "cellmetadata": {
-      "bytelimit": 2048000,
-      "rowlimit": 10000
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
      },
-     "inputwidgets": {},
+     "inputWidgets": {},
      "nuid": "67f2404c-a6b6-4342-9dac-638a2bd7731c",
-     "showtitle": false,
+     "showTitle": false,
      "title": ""
     }
    },
    "outputs": [],
    "source": [
     "import base64\n",
     "import os\n",
-    "cert_file= open(\"/workspace/users/fabianm@microsoft.com/fabianm-spark-auth-sp-cert.pem\",\"rb\")\n",
+    "\n",
+    "\n",
+    "cert_file= open(\"/Workspace/Users/fabianm@microsoft.com/fabianm-spark-auth-sp-cert.pem\",\"rb\")\n",
```

Comment:
FYI, this shares your work email address as part of the sample, right?

Reply:
Not the end of the world - but fixed.